api/app/Models/Team.php

```diff
@@ -25,4 +29,9 @@ class Team extends LaratrustTeam
     ];
 
     public $guarded = [];
+
+    public function departments(): BelongsToMany
+    {
+        return $this->belongsToMany(Department::class, 'team_department');
+    }
```

Comment:
Is this right? A department can belong to more than one team? 🤔 I'm not aware of the requirements but this seems a little odd to me.

Reply:
I just mirrored how another pivot table was represented in the model files

Reply:
@tristan-orourke What should this relationship look like?

- One to One (how it appears in the table view)
- One to Many (how it appears in the proposed graphql model)
- Many to Many (how it is defined here)

Reply:
I just confirmed with @Jerryescandon - it should be Many to Many. One team can cover multiple depts, and there's no restriction on multiple teams being in the same dept.